prime-router/src/main/kotlin/fhirengine/engine/QueueMessage.kt

```diff
@@ -81,7 +82,16 @@ interface ReportIdentifyingInformation {
     val topic: Topic
 }
 
-abstract class ReportPipelineMessage : ReportIdentifyingInformation, WithDownloadableReport, QueueMessage()
+interface OriginalMessageInformation {
```

Comment:
I see value in passing the original report ID to every step for purposes of lineage tracking via logs, less so for the original format. How difficult would it be to put both in a class called "OriginalMessageInfo", that way we can expand this class later if we want without having to explicitly alter all these classes every time? We also wouldn't have to explicitly pass originalReportFormat to two functions that don't care about it just to get it to translate, but if we wrap it in a class like OriginalMessageInfo maybe it makes more sense?